src/transformers/models/layoutlmv3/processing_layoutlmv3.py

```diff
@@ -100,22 +118,31 @@ def __call__(
 
         Please refer to the docstring of the above two methods for more information.
         """
+        output_kwargs = self._merge_kwargs(
+            LayoutLMv3ProcessorKwargs,
+            tokenizer_init_kwargs=self.tokenizer.init_kwargs,
+            image_processor_init_kwargs=self.image_processor.init_kwargs,
```

Comment:
@zucchini-nlp @NielsRogge 

I may have found a but with the CI: no tests in [tests_exotic_models](https://app.circleci.com/pipelines/github/huggingface/transformers/101039/workflows/c71c8dcf-b06d-4918-8c4d-4769621e500f/jobs/1345351) ever gets run

E.g. the tests should fail because of this line here ^^. But the tests passed anyway..

@NielsRogge are the layoutlmv3 models deprecated?